caffe2/python/operator_test/upsample_op_test.py

```diff
@@ -40,52 +40,72 @@ def test_upsample(self, height_scale, width_scale, height, width,
                      gc, dc):
 
         np.random.seed(seed)
-        op = core.CreateOperator(
-            "UpsampleBilinear",
-            ["X"],
-            ["Y"],
-            width_scale=width_scale,
-            height_scale=height_scale,
-        )
 
         X = np.random.rand(
             batch_size, num_channels, height, width).astype(np.float32)
-
-        def ref(X):
-            output_height = np.int32(height * height_scale)
-            output_width = np.int32(width * width_scale)
-
-            Y = np.random.rand(
-                batch_size, num_channels, output_height,
-                output_width).astype(np.float32)
-
-            rheight = ((height - 1) / (output_height - 1)
-                    if output_height > 1
-                    else float(0))
-            rwidth = (width - 1) / (output_width - 1) if output_width > 1 else float(0)
-
-            for i in range(output_height):
-                h1r = rheight * i
-                h1 = int(h1r)
-                h1p = 1 if h1 < height - 1 else 0
-                h1lambda = h1r - h1
-                h0lambda = float(1) - h1lambda
-                for j in range(output_width):
-                    w1r = rwidth * j
-                    w1 = int(w1r)
-                    w1p = 1 if w1 < width - 1 else 0
-                    w1lambda = w1r - w1
-                    w0lambda = float(1) - w1lambda
-                    Y[:, :, i, j] = (h0lambda * (w0lambda * X[:, :, h1, w1] +
-                        w1lambda * X[:, :, h1, w1 + w1p]) +
-                        h1lambda * (w0lambda * X[:, :, h1 + h1p, w1] +
-                        w1lambda * X[:, :, h1 + h1p, w1 + w1p]))
-
-            return Y,
-
-        self.assertReferenceChecks(gc, op, [X], ref)
-        self.assertDeviceChecks(dc, op, [X], [0])
-        self.assertGradientChecks(gc, op, [X], 0, [0], stepsize=0.1, threshold=1e-2)
+        scales = np.array([height_scale, width_scale]).astype(np.float32)
+
+        ops = [
+            (
+                core.CreateOperator(
```

Comment:
Lint the code? the format looks a little bit weird here.

Reply:
Linted